content/en/docs/setup/upgrade/index.md

```diff
@@ -56,18 +56,24 @@ istio-sidecar-injector-canary   2020-04-28T19:03:26Z
 
 ### Data plane
 
-Simply installing the new revision has no impact on the existing proxies. To upgrade these,
-you must configure them to point to the new control plane. This is controlled during sidecar injection
+Installing the new revision has in place upgraded the istio-ingress gateway pod and service, which are automatically configured to point to the 'istiod-canary' control plane.
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system -l app=istio-ingressgateway
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-7bf8767888-9cchz   1/1     Running   0          8m30s
+{{< /text >}}
+
```

Comment:
What's the purpose of this command? It doesn't seem to show anything to indicate "in place upgrade" has or will happen.
```suggestion
```

Reply:
The purpose is to show it doesn't have a canary next to it, e.g. old one is removed and this new one is deployed.

Reply:
Let me know - happy to remove it if there is no need to show it.

Reply:
Yes, I would suggest removing, but I think you need to add some text to explain this better. Something that explains that unlike istiod, gateways do not have revisions, but are updated to the latest revision, immediately when a new istiod revision is deployed. How does it know what is the latest revision tag to use?